src/FOM/CoreBundle/Component/ExportResponse.php

```diff
@@ -158,6 +179,7 @@ public function setType($type)
                 $this->headers->add(array('Content-Type' => 'text/csv;charset=' . self::UTF_16_LE));
                 break;
             case self::TYPE_XLS:
+            case self::TYPE_XLSX:
```

Comment:
according to https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types the MIME type for XLSX is `application/vnd.openxmlformats-officedocument.spreadsheetml.sheet`